### ISSUE TITLE
Error running installer on Node v9

### ISSUE THREAD

**Issue by juliandavidmr** [raphamorim/react-tv#101]
_react-tv-cli_ does not install. An error associated with the node version occurs:
```bash
error react-tv-cli@0.3.0: The engine "node" is incompatible with this module. Expected version ">=8".
error Found incompatible module
```

**The problem is that it is running on Node v9**

- [x] Linux Opensuse Leap
- [x] Node v9.3.0
- [x] Command tested: `sudo yarn global add react-tv-cli`

**Comment by raphamorim** [comment]
Yeah, `react-tv-cli` requires a nodejs `<=8.9`. 
Because some dependency on renderer (for rollup) doesn't work with nodejsv9. 
However now renderer and CLI are separated packages. Make sense to change it. :octocat:

**Comment by raphamorim** [comment]
Hi @juliandavidmr, sorry by delay.
I published `0.3.1` today. This version should make it work. Can you check it? :octocat:

**Comment by raphamorim** [comment]
I'm closing this issue, any isssue we can reopen it.
